intel_owl/settings.py

```diff
@@ -219,7 +219,6 @@
     "formatters": {
         "stdfmt": {
             "format": "%(asctime)s - %(name)s - %(funcName)s - %(levelname)s - %(message)s",
-            "datefmt": "%Y-%m-%d %H:%M:%S",
```

Comment:
:/

Reply:
this is a commit of mine too. This is to add milliseconds in logs